File: sample.r
library(subprocess)

input<-file('stdin', 'r')

get_next_data <- function() {
  readLines(input, n=1)
}

submit_prediction <- function(pred) {
  cat(pred + "\n")
}

### YOUR CODE BELOW

### Basic sample algorithm
while (TRUE) {
  ### Read data
  data <- get_next_data()
  
  ### Guess 3 every time
  submit_prediction(3) 
}

File: sample.py
import time
import subprocess
import sys

def get_next_data():
    """
    Reads input from standard input

    Use this to supply your model with input
    Input will not be supplied until output is 
    generated for the previous input
    """
    return input()

def submit_prediction(prediction):
    """
    Submits your prediction to standard output
    """
    print(str(prediction))

def eprint(msg):
    """
    Prints to standard error

    Use this to debug / develop. 
    This output will not be scored
    """
    print(msg, file=sys.stderr)

### YOUR CODE BELOW

while(True):
    try:
        ### Get next line of data
        data = get_next_data()

        ### Guess 0 every time
        submit_prediction(0)
    except EOFError as e:
        break
